load.Rmd

```diff
@@ -641,7 +638,7 @@ if(!"gendereq_slc" %in% scens)
 # Rename columns
 l@data <- rename(l@data, Home = Area.of.residence, Work = Area.of.workplace)
 names(zones@data)
-
+l$clc_perc <- l$clc * 100
```

Comment:
Not really - just for consistency. Happy for you to remove it and multiply in pct-shiny. Note that it means your Current level of cycling (%) in the lines tab is currently wrong. My thinking was to replace a call to `l$clc` with `l$lclc_perc` but if there's a better way, go with it. The key thing I wanted to avoid was breaking existing code and I think I've managed that in any case.